React/Executors/RCTJSCExecutor.m

```diff
@@ -314,7 +316,16 @@ - (void)setUp
   }];
 
   [self executeBlockOnJavaScriptQueue:^{
-    RCTInstallJSCProfiler(_bridge, self.context.ctx);
+    RCTJSCExecutor *strongSelf = weakSelf;
+    if (!strongSelf.valid) {
+      return;
+    }
+
+    JSContext *context = strongSelf.context.context;
+    RCTInstallJSCProfiler(_bridge, context.JSGlobalContextRef);
+
+    [[NSNotificationCenter defaultCenter] postNotificationName:RCTJavaScriptContextCreatedNotification
```

Comment:
Yep, that was pretty major mistake on my part. I realized that a little while after this was merged. 😞